### PR TITLE
New-brand-xl-template.md

### PR DESCRIPTION
**Pull request by fabicastp**
Adding XL brand template

### CODE REVIEW

File: content/departments/engineering/design/templates/brand-XL-template.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  - Estimate: More than a month
+  - Weighting factors:
+    - Note: a project of this size requires resources that increase its risk profile.
+    - [HIGH|MEDIUM] risk
+    - [HIGH|MEDIUM|LOW] clarity
+
+  ### Discover
+  Extra large projects should entail more comprehensive research efforts.
+  -[ ] Project brief doc
```
Comment (rrhyne) [inline content/departments/engineering/design/templates/brand-XL-template.md]: Design would not be responsible for the entire project brief. 

Can we update to contribute to the brief and outline the specific sections design would be responsible for?

File: content/departments/engineering/design/templates/brand-XL-template.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    -[ ] Who, What, Why
+    -[ ] Constraints
+    -[ ] Constraints (what we can't do)
+    -[ ]	Deliverables (site nav, mood board, content strategy)
+    -[ ]Timeline
+    -[ ]Distribution channels
+    -[ ]Creative direction
+    -[ ]Resources
+  -[ ] Relevant feedback (internal/external)
+  -[ ] Site analytics
```
Comment (rrhyne) [inline content/departments/engineering/design/templates/brand-XL-template.md]: I believe the points above are a PMM or PM responsibility, but design does have some role in feedback, such as user testing results. 

Can we update to include only what design is responsible for?

File: content/departments/engineering/design/templates/brand-XL-template.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  -[ ] Analogous/competitive research - Link
+
+  ### Define
```
Comment (rrhyne) [inline content/departments/engineering/design/templates/brand-XL-template.md:34]: Where does defining and arriving at a consensus on creative direction live? I'm talking about mood boards and the like.

File: content/departments/engineering/design/templates/brand-XL-template.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  -[ ] Findings, recommendations and direction
+  -[ ] GitHub issue
+
+  ### Design
+  -[ ] Site architecture/User journey
+    -[ ] Review with stakeholders
+  -[ ] Wireframes
+	  -[ ] Review with stakeholders
+    -[ ] Designer review
+  -[ ] Mockup (desktop and mobile)
```
Comment (rrhyne) [inline content/departments/engineering/design/templates/brand-XL-template.md]: This probably looks a bit more like:

```suggestion
  -[ ] Concepts
    - [ ] Stakeholder review
  - [ ] Direction from concepts
      - [ ] Stakeholder review
  - [ ] Mobile version
```

File: content/departments/engineering/design/templates/brand-XL-template.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    -[ ] Review with stakeholders
+    -[ ] Designer review
+  -[ ] Prototype - Link
```
Comment (rrhyne) [inline content/departments/engineering/design/templates/brand-XL-template.md:52]: A good way to mark optional items:

```suggestion
  -[ ] Prototype (optional) - Link
```

Comment (traceyljohnson) [reply content/departments/engineering/design/templates/brand-XL-template.md:52]: @rrhyne I think for an XL, we would definitely want a prototype, right?

File: content/departments/engineering/design/templates/brand-XL-template.md (hunk 3 above)
Comment (traceyljohnson) [reply content/departments/engineering/design/templates/brand-XL-template.md:39]: I would think it goes between wireframes and mockup. Maybe we do:
-[  ] Moodboarding/Initial concept options
   - [ ] Stakeholder review
- [ ]  Mockups based on approved concept
      - [ ] Stakeholder review
- [ ] Mobile version

Comment (traceyljohnson) [reply content/departments/engineering/design/templates/brand-XL-template.md:34]: I think with this size of project, it'd be good to do the moodboards and even a few initial concepts (half-page conceptual mocks) early on. During the Define stage.

**Review by traceyljohnson — APPROVED** [review]

**Pull request merged**